.github/workflows/test-installers.yml

```
@@ -177,8 +176,8 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: 20
-          cache: "npm"
+          node-version: 22
```

If we're pinning to a specific version everywhere else, we might as well want to do that here? 🤔

```suggestion
          node-version: 22.15.1
```

---

I like the sound of that, also makes the update next time easier when everything aligns so we know we didn't miss anything. Didn't wrap it into this pr but I'll get in a drive by or something as a follow up. Got this one out quicker to unblock things.